nirc_ehr/resources/etls/histopathology.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<etl xmlns="http://labkey.org/etl/xml">
+    <name>Histopathology</name>
+    <description>Histopathology data</description>
+    <transforms>
+        <transform id="step1" type="org.labkey.di.pipeline.TransformTask">
+            <description>Copy to histopathology target</description>
+            <source schemaName="dbo" queryName="q_histopathology"/>
+            <destination schemaName="study" queryName="histopathology"
+                         bulkLoad="true" batchSize="5000" targetOption="merge">
```

Comment:
I don't think these are very long ETLs. I would suggest just using targetOption="truncate". That will ensure to also delete any records that were deleted from the source and ensures there are no merge issues.

Reply:
Ok, sounds good